<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BSAFilesToPack</class>
 <widget class="QWidget" name="BSAFilesToPack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="UncompressableFilesLabel">
     <property name="text">
      <string>Cannot be compressed</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QListWidget" name="TexturesFilesListWidget"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="TexturesFilesLabel">
     <property name="text">
      <string>Textures</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="StandardFilesListWidget"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="StandardFilesLabel">
     <property name="text">
      <string>Standard</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QListWidget" name="UncompressableFilesListWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="StandardFilesEditButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="TexturesFilesEditButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="UncompressableFilesEditButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
